src/components/Input/Input.scss

```diff
@@ -0,0 +1,22 @@
+
+
+.input {
+    font-size: 16px;
+    line-height: 24px;
+    padding: 4px 8px;
+    border: 1px solid #9B9B9B;
```

Comment:
Let's replace all the hexcodes with sass variables in the `common/variables` file for consistency